axelrod/strategies/resurrection.py

```diff
@@ -34,3 +34,33 @@ def strategy(self, opponent: Player) -> Action:
         else:
             return opponent.history[-1]
 
+class DoubleResurrection(Player):
+    """
+    A player starts by cooperating and defects if the number of rounds 
+    played by the player is greater than five and the last five rounds are cooperations.
+    If the last file rounds were defections, the player cooperates.
+    
+    Names:
+    - DoubleResurrection: Name from CoopSim https://github.com/jecki/CoopSim     
+    """
+
+    name = 'DoubleResurrection'
+    classifier = {
+        'memory_depth': 1,  
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent: Player) -> Action:
+        if len(self.history) == 0:
+            return C
+        if len(self.history) >= 5 and self.history[-5:] == [C, C, C, C, C]:
+            return D
+        elif len(self.history) >= 5 and self.history[-5:] == [D, D, D, D, D]:
+            return C
+        else:
+            return opponent.history[-1]
```

Comment:
Another way to do this would be to use an 'actions' dictionary rather than conditionals. Feel free to ignore this suggestion, but I always prefers using dicts to contain logic whenever possible!
```
def strategy(self, opponent: Player) -> Action:
    if not self.history:
        return C

    is_history_sufficient = len(self.history) >= 5    
    actions = {
        (True, 'CCCCC'): 'D',
        (True, 'DDDDD'): 'C'
    }

    try:
        return actions[(is_history_sufficient, self.history[-5:])]
    except KeyError:
            return opponent.history[-1]
```